lib/widgets/chart.dart

```diff
@@ -23,11 +23,14 @@ class Chart extends StatelessWidget {
         }
       }
 
+      print(DateFormat.E().format(weekDay),);
```

Comment:
remove comma if allowed.

Reply:
@MapleSyyrup can you remove the comma if allowed and will not cause any error in this line?